### PR TITLE
Fix timing screen.

### PR DESCRIPTION
**Pull request by roberthoenig**
@borisyankov FYI. This does not fix the wrong time duration, only the wrong timestamp. I'm looking into the duration now.

### CODE REVIEW

**Comment by roberthoenig** [comment]
Hmm, tried to reproduce the duration issue on both the emulator and a real device -- without success. Maybe it's been resolved by some other commit since the latest test deploy?

**Comment by borisyankov** [comment]
Hmm not that likely.

File: src/utils/timing.js [hunk 1]
```diff
@@ -16,15 +15,6 @@ const start = (key: string) => {
   timingMap[key] = now();
 };
 
-const startGroup = (key: string) => {
-  timingMap[key] = now();
```
Comment (borisyankov) [inline src/utils/timing.js]: That might explain some perf underestimations on the HTML parsing and rendering in the native HTML. And will explain the higher than expected perf boost when switching to WebView ;)

**Comment by roberthoenig** [comment]
Okay, ignore the previous comment. I now **did** manage to reproduce the way-too-small-duration issue: The problem is (again) that we divided Date.now() by 1000. So when the time 1.56 ms is displayed, it should actually be 1560 ms. This did not show up in the emulator when the emulator used remote JS debugging, because in that case, it used performance.now() instead of Date.now(). performance.now() works fine for duration, it just cannot be used for determining the current time. So, in the end, this PR should also fix the way-too-small-duration issue.

**Comment by borisyankov** [comment]
I'm cool with removing the group timings. We can re add them later if we need to (with a proper fix).
They were used to give aggregate timing on functiona calls that on their own did take little time but were called often (like the HTML parsing function of past times).

File: src/utils/timing.js [hunk 2]
```diff
@@ -2,27 +2,32 @@
...
 
 const timingMap: { [string]: number } = {};
+const preciseTimingMap: { [string]: number } = {};
 const log: TimingItemType[] = [];
 
-const now = (): number =>
-  typeof performance !== 'undefined' ? performance.now() : Date.now() / 1000;
+const now = (): number => Date.now();
+
+const preciseNow = (): number =>
```
Comment (borisyankov) [inline src/utils/timing.js]: A non terribe idea would have been to do that:
```js
const performanceNow = require('fbjs/lib/performanceNow');
```
Or copy-paste the very simple polyfill directly from:
https://github.com/facebook/fbjs/blob/master/packages/fbjs/src/performance/performanceNow.js

But scratch that as Meltdown or Spectre fixes decrease the resolution in browsers to 1-2msec. We should just use `Date.now()` and be mindful of the low precision.

File: src/utils/timing.js [hunk 3]
```diff
@@ -2,27 +2,32 @@
...
+  typeof performance !== 'undefined' ? performance.now() : Date.now();
 
 const add = (item: TimingItemType) => {
   log.push(item);
 };
 
 const start = (key: string) => {
   timingMap[key] = now();
+  preciseTimingMap[key] = preciseNow();
```
Comment (borisyankov) [inline src/utils/timing.js]: If we had a precise timer, I would consider this suboptimal - we might have preferred a single map with objects as keys. Seeing we are not keeping the precision, I think we can not use that at all.

**Comment by roberthoenig** [comment]
Closing; @gnprice merged both commits into master.

**Pull request closed without merging**